impress-org/givewp
# Epic Feature: Column sorting UI

## Description
This PR contains changes made that update our current react tables and how they operate. As the moment, the changes in this PR will only work for the Donations table.

Main Changes include : 
- Ability for tables to receive column data from the window instead of statically through configs.
- Dynamic aria-sort support for column sorting.
- Additional table query arguments : sortDirection, sortColumn, browser locale.
- Interweave package to handle SSR templates
- Sorting component 
- Components ListTable directory has been refactored with sub-directories. (Previously had 30+ loose files)

## Affects

Donor, Donation & Donor Form tables.

## Visuals


https://user-images.githubusercontent.com/75056371/200430637-179dce3f-35ce-4d76-bf87-d6a94d0af76a.mov


## Testing Instructions
- View Donations > Donations tab.
- Verify sorting for all sortable columns work as intended.
- Verify all bulk actions work work as intended.
- Verify Search filtering works as intended.
## Pre-review Checklist
